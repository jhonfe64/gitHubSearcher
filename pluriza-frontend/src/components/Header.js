/*
    Este componente Header es un componente sin estadoq ue solo muestar el logo y alfo de info para el ancabezado
*/


import React from 'react';
import  HeaderElement from '../elements/HeaderElement';

function Hearder() {
    return (
        <HeaderElement className="container">
            <img src="../images/logo.png" alt="logo" />
            <div className="textContainer">
                <h1>GitHub Searcher</h1>
                <h3>Search users or repositories below</h3>
            </div>
        </HeaderElement>
           
    )
}

export default Hearder
