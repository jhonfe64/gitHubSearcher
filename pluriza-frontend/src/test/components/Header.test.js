
import React from 'react';
import {shallow} from 'enzyme';
import Header from '../../components/Header';


describe('Pruebas sobre el compoenente <Header />', ()=>{
    test('probar que el componente se rederize correctamente', ()=>{
     
        const wrapper = shallow(< Header />);
        expect(wrapper).toMatchSnapshot();
    })
})