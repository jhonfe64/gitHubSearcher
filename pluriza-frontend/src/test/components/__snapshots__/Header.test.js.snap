// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas sobre el compoenente <Header /> probar que el componente se rederize correctamente 1`] = `
<styled.div
  className="container"
>
  <img
    alt="logo"
    src="../images/logo.png"
  />
  <div
    className="textContainer"
  >
    <h1>
      GitHub Searcher
    </h1>
    <h3>
      Search users or repositories below
    </h3>
  </div>
</styled.div>
`;
